Jenkinsfile

```diff
@@ -197,28 +203,17 @@ pipeline {
                     environment {
                         GIT_CACHE_PATH = "${HOME}/cache"
                         SCCACHE_BUCKET = credentials("brave-browser-sccache-mac-s3-bucket")
-                        PATH = "${HOME}/.cargo/bin:${PATH}:/usr/local/bin/"
                     }
                     stages {
-                        stage("unlock") {
-                            steps {
-                                sh "rm -rf ${GIT_CACHE_PATH}/*.lock"
-                            }
-                        }
                         stage("checkout") {
                             when {
-                                expression { return !fileExists("package.json") }
-                            }
-                            steps {
-                                git branch: "${BRANCH_TO_BUILD}", url: "https://github.com/brave/brave-browser.git"
-                            }
-                        }
-                        stage("clean") {
-                            when {
-                                expression { params.CLEAN_WORKSPACE }
+                                anyOf {
+                                    expression { params.WIPE_WORKSPACE }
+                                    expression { return !fileExists("package.json") }
+                                }
                             }
                             steps {
-                                sh "git clean -ffxd"
+                                checkout([$class: 'GitSCM', branches: [[name: "${BRANCH_TO_BUILD}"]], extensions: [[$class: 'WipeWorkspace']], userRemoteConfigs: [[url: 'https://github.com/brave/brave-browser.git']]])
```

Comment:
What does `userRemoteConfigs` specify? I can't find documentation on it.

Reply:
specifies the remote/repo